api/utils/api-url.js

```diff
@@ -0,0 +1,5 @@
+const apiConfig = {
```

Comment:
This should be in tests folder. 

also, this should be like 

```
export const API_URL='https://localhost:2015'
```

Reply:
Alright.